scripts/xe-edit-bootloader

```diff
@@ -15,7 +15,7 @@ XE="@BINDIR@/xe"
 
 device_number=
 grub_conf=
-default_file_list="/boot/grub/menu.lst /grub/menu.lst /boot/grub/grub.cfg /grub/grub.cfg"
+default_file_list="/boot/grub/menu.lst /grub/menu.lst /boot/grub/grub.cfg /grub/grub.cfg /boot/grub2/grub.cfg /grub2/grub.cfg"
```

Comment:
If this list is going to be expanded, we should include the whole list from pygrub:
https://github.com/mirage/xen/blob/master/tools/pygrub/src/pygrub#L428
We should also make sure the list is in the same order (so the correct config file will be used if multiple config files are found)
We should therefore use:
default_file_list="/boot/grub/grub.cfg /grub/grub.cfg /boot/grub2/grub.cfg /grub2/grub.cfg"
default_file_list="$default_file_list /boot/isolinux/isolinux.cfg /boot/extlinux/extlinux.conf /boot/extlinux.conf /extlinux/extlinux.conf /extlinux.conf"
default_file_list="$default_file_list /boot/grub/menu.lst /boot/grub/grub.conf /grub/menu.lst"